src/ServerDaemon.chpl

```diff
@@ -401,7 +401,11 @@ module ServerDaemon {
     
             this.connectUrl = this.getConnectUrl(this.serverToken);
             this.createServerConnectionInfo();
-            this.printServerSplashMessage(this.serverToken,this.arkDirectory);
+            if serverInfoNoSplash {
+                writeln(getConfig());
```

Comment:
After this `writeln`, you need a `stdout.flush()`. Without this you cannot read the output JSON on the Python side. Since the process is non-terminating when running Arkouda, it appears to just hang in the current configuration.

Reply:
Fixed!